### PR TITLE
user uri interest score

### PR DESCRIPTION
**Pull request by yingjieMiao**
@stkem 

this completes the pipeline. Basic cosine distance score. No data scaling yet, as this requires some global statistics. We can tweak this later.

### CODE REVIEW

File: kifi-backend/modules/cortex/app/com/keepit/common/commanders/LDACommander.scala [hunk 1]
```diff
@@ -69,4 +71,16 @@ class LDACommander @Inject() (
       userTopicRepo.getTopicMeanByUser(userId, wordRep.version)
     }
   }
+
+  def userUriInterest(userId: Id[User], uriId: Id[NormalizedURI]): Float = {
+    val MIN_SCORE = 0.0001f
```
Comment (stkem) [inline kifi-backend/modules/cortex/app/com/keepit/common/commanders/LDACommander.scala]: What is this for?

Comment (yingjieMiao) [inline kifi-backend/modules/cortex/app/com/keepit/common/commanders/LDACommander.scala]: a default value, e.g. when user or uri's topic is not available. could be 0. depends on how the final formula looks like.

Comment (stkem) [inline kifi-backend/modules/cortex/app/com/keepit/common/commanders/LDACommander.scala]: I thought this is cosine similarity, which is between -1 and 1, so aren't cutting off over half of the score range?

**Comment by yingjieMiao** [comment]
@stkem  FYI
updated with `Option[Float]`

**Comment by stkem** [comment]
:+1:

**Pull request merged**